examples/user_guide/Plotting_API.ipynb

```diff
@@ -0,0 +1,643 @@
+    "\n",
+    "For additional information about working and plotting with HoloViews see the [User Guide](http://holoviews.org/user_guide/index.html), as we will focus on using the high-level plotting API in this overview and skip most of the mechanics going on behind the scenes.\n",
+    "\n",
+    "We will be focusing on three different datasets:\n",
+    "\n",
+    "- A small CSV file of US State level crime data\n",
+    "- A parquet file of airline data"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "crime = intake.cat.us_crime\n",
+    "flights = intake.cat.airline_flights"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## The plot interface"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The ``DataSource.plot`` interface provides a powerful high-level API to generate complex plots. The ``.plot`` API can be called directly or used as a namespace to generate specific plot types."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### The plot method"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The simplest way to use the plotting API is to specify the names of columns to plot on the ``x``- and ``y``-axis respectively:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "crime.plot(x='Year', y='Violent Crime rate')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "An additional convenience on top of this explicit API is to specify an additional ``by`` variable, which groups the data by one or more additional columns. As an example here we will plot the departure delay ('depdelay') as a function of 'distance' and group the data by the 'carrier' (and then select just a subset of carriers):"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "flights.plot(x='distance', y='depdelay', by='carrier', kind='scatter', alpha=0.2).select(carrier=[b'OH', b'F9'])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This approach differs slightly from the usual pandas plotting API, which usually assumes that the dataframe has a meaningful index and the columns represent similar observations. We can reproduce this functionality by enabling the ``use_index`` option and declaring the columns to plot (by default it will use all columns). The ``use_index`` can either be set to ``True`` in which case it will use the actual index or reference a column in which case it will be treated like the index. In this approach the y-axis will receive a default label named 'value' however we can already override the label using the ``value_label`` option.\n",
+    "\n",
```

Comment:
Shouldn't use_index be True by default, so that people who use Pandas can use Pandas's mechanisms to declare which dimensions are the index(es) (key dimensions)?

Reply:
I think it is True now actually.